scripts/package_chrome_extension.sh

```diff
@@ -16,9 +16,16 @@ cd tmp
 zip -r ../build/kiwix-chrome-unsigned-extension-$VERSION.zip www webextension manifest.json LICENSE-GPLv3.txt service-worker.js README.md
 cd ..
 if [ "${TAG}zz" == "zz" ]; then
-    # Package the extension with Chromium, if we're not packaging a public version
-    echo "Signing the extension for Chrome with a local Chromium, version $VERSION"
-    chromium-browser --no-sandbox --pack-extension=tmp --pack-extension-key=./scripts/kiwix-html5.pem
+    # Package the extension with Chrome or Chromium, if we're not packaging a public version
+    if hash chromium-browser 2>/dev/null then
+        echo "Chromium is available"
+        CHROME_BIN=chromium-browser
+    else
+        echo "Chrome is available"
+        CHROME_BIN=chrome
+    fi
+    echo "Signing the extension for $CHROME_BIN, version $VERSION"
+    $CHROME_BIN --no-sandbox --pack-extension=tmp --pack-extension-key=./scripts/kiwix-html5.pem
```

Comment:
@mossroy Please check that the above code looks correct.

I'm also a bit worried about the single `.` before `/scripts` in the line above: `--pack-extension-key=./scripts/kiwix-html5.pem`. By analogy with the firefox packaging file, it would seem this might need to be `../scripts` (but it was originally like this, so I'm probably missing something).

Reply:
Ah, also, kiwix-html5.pem is a secret file not committed to the repo, right? In which case we'll have to make it too and upload it as a secret. Correct?

Reply:
I've added it as CHROME_EXTENSION_KEY and have uploaded the secret. I need to find out if --pack-extension-key can accept the constant, or if we need to make a file.

Reply:
Seems it only takes a file. Will make one.

Reply:
I've made the file from the secret. My remaining doubt is about whether to use `../scripts` or `./scripts` here. Is the location read from the current working directory (in which case I think we need `../scripts`) or from the perspective of the chrome executable (in which case we may need something more elaborate that includes the $GITHUB_WORKSPACE directory structure)?

Reply:
The `./scripts` is probably OK because there is a `cd ..` above.
These scripts would deserve a good refactoring : the bash coding style could really be improved

You're right to create this new secret for Chrome. You'll probably have to serialize it to a file, like for the ssh key

Reply:
Thanks. I've added code to make the file already. I can run a tests from this branch. It will upload to nightly, but should allow us to see if things are working in the "nightly" mode.